raiden-dapp/tests/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "..",
```

Comment:
Why? :thinking:

Reply:
So it bases on the root of the package (the same as the upper tsconfig). Required in order to properly reference on the `paths` alias. Maybe it'd work with the full relative path there as well, but this is more common, and also how it's done in the SDK.